.changeset/gold-trains-sniff.md

```diff
@@ -0,0 +1,5 @@
+---
+"medusa-plugin-meilisearch": patch
```

Comment:
suggestion: should this be a minor/major?

Thinking if people have `"^x.x.x"` in their deps + an old version of Meilisearch running; this will likely break their environment

Reply:
Nice catch! Would make sense to publish a minor for this one 👍